File: src/main/java/com/sudosoo/takeiteasy/TakeItEasyApplication.java
package com.sudosoo.takeiteasy;

import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@SpringBootApplication
@Configuration
@EnableJpaAuditing
@EnableTransactionManagement
@EnableBatchProcessing
public class TakeItEasyApplication {

    public static void main(String[] args) {
        SpringApplication.run(TakeItEasyApplication.class, args);
    }

}

File: boot/src/main/kotlin/com/sudosoo/takeItEasy/TakeItEasyApplication.kt
package com.sudosoo.takeItEasy

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.EnableAspectJAutoProxy
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@EnableJpaAuditing
@EnableAspectJAutoProxy
@SpringBootApplication
class TakeItEasyApplication
    fun main(args: Array<String>) {
        runApplication<TakeItEasyApplication>(*args)
    }
